lib/screens/authentication/import_key/components/import_key_accounts_widget.dart

```diff
@@ -24,42 +24,40 @@ class ImportKeyAccountsWidget extends StatelessWidget {
             return const SizedBox.shrink();
           case PageState.success:
             return ListView(
+                shrinkWrap: true,
                 children: state.accounts
-                    .map((ProfileModel? profile) => Padding(
-                          padding: const EdgeInsets.all(16),
```

Comment:
No logic changes, 
Only real changes here are a padding removed. and added  shrinkWrap: true, 
thats all.